### ISSUE TITLE
Unrecognized field "Names" (Class com.kpelykh.docker.client.model.Container)

### ISSUE THREAD

**Issue by lfryc** [kpelykh/docker-java#17]
I'm getting following error (shortened):

```
UnrecognizedPropertyException: Unrecognized field "Names" (Class com.kpelykh.docker.client.model.Container), not marked as ignorable
 at [ByteArrayInputStream; line: 1, column: 226] (through reference chain: com.kpelykh.docker.client.model.Container["Names"])
```

with the request/response:

```
INFO hudson.slaves.NodeProvisioner update
Image of jenkins-centos-1 provisioningE successfully completed. We have now 1 computer(s)
INFO com.nirima.jenkins.plugins.docker.DockerCloud provision
Excess workload after pending Spot instances: 1
INFO com.sun.jersey.api.client.filter.LoggingFilter log
19 * Client out-bound request
19 > GET http://localhost:4243//v1.8/containers/json?limit=-1&before=&since=&all=0&size=0
19 > Accept: application/json

INFO com.sun.jersey.api.client.filter.LoggingFilter log
19 * Client in-bound response
19 < 200
19 < Content-Type: application/json
19 < Content-Length: 249
19 < Date: Fri, 31 Jan 2014 16:57:34 GMT
19 < 
[{"Id":"c1bc8e3e2cd45b2a2561f494a0920e9f12956d1563718e84a4e81de404bfa62a","Image":"jenkins-centos-1:latest","Command":"/bin/bash ","Created":1391186016,"Status":"Up 23 minutes","Ports":null,"SizeRw":0,"SizeRootFs":0,"Names":["/agitated_albattani"]}]
```

whole stacktrace:
https://gist.github.com/lfryc/929086d704418e00d8b1

**Comment by lfryc** [comment]
I believe this is incompatibility with API v1.8, I'm running Docker 1.7.2.

**Comment by kpelykh** [comment]
fixed in docker-java  0.7.6-SNAPSHOT
